easybuild/easyconfigs/a/AdaptiveCpp/AdaptiveCpp-23.10.0-cpeGNU-22.12-rocm-5.2.3.eb

```diff
@@ -0,0 +1,86 @@
+easyblock = 'CMakeMake'
+
+name =    'AdaptiveCpp'
+version = '23.10.0'
+versionsuffix = '-rocm-5.2.3'
+
+homepage = 'https://adaptivecpp.github.io/'
+
+whatis = [
+    "Description: AdaptiveCPP is a modern SYCL implementation targeting CPUs and GPUs, with a focus on leveraging existing toolchains such as CUDA or HIP"
+]
+
+description = """
+hipSYCL is a modern SYCL implementation targeting CPUs and
+GPUs, with a focus on leveraging existing toolchains such as CUDA or HIP.
+
+It targets any CPU via OpenMP, NVIDIA GPUs via CUDA, AMD GPUs via HIP/ROCm and 
+Intel GPUs via oneAPI Level Zero and SPIR-V (experimental). However, this 
```

Comment:
This description is outdated. See the github repository for how the project positions itself. Intel GPU support is non-experimental, and can also go through OpenCL, not just Level Zero.